### ISSUE TITLE
allow markdown translations/notes etc.

### ISSUE THREAD

**Issue by bhajneet** [shabados/database#1819]
I would like to be able to indicate italicised / bolded words in translations and notes using markdown in the DB. Other developers can strip these characters if they don't want to use the information.

**Comment by dilrajsachdev** [comment]
I think this is with formatting for end user, so have to be apply on the front-end and not in the DB

**Comment by Harjot1Singh** [comment]
Makes sense. I ask in case we ever want to add a markdown validator to the schema / collections validator in the future.
